src/components/MonitoringBtn.jsx

```diff
@@ -1,9 +1,99 @@
-import React from 'react'
 
-const MonitoringBtn = () => {
+
```

Comment:
don't write buisness logic on app.jsx. and try to make several components if needed. You can move canvas to a different component.